src/SOS/SOS.UnitTests/SOS.UnitTests.csproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="$(RepositoryEngineeringDir)\InstallRuntimes.proj" />
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
Out of curiosity, was this needed or just goodness?

Reply:
It is needed by private builds and now in my future arm/arm64 fixes (doesn't install 2.1).